adsingestp/parsers/jats.py

```diff
@@ -896,6 +897,11 @@ def _parse_page(self):
                     self.article_meta.find("counts").find("page-count").get("count", "")
                 )
 
+    def _parse_fulltext(self):
+        if self.body is not None:
+            body_text = str(self.body.extract())
+            self.base_metadata["fulltext"] = {"body": body_text, "language": None}
```

Comment:
is `language` always going to be set to None for now? Do we have a way of determining the language from the input xml files?

Reply:
I would like to keep it as None for now, at least for JATS.  I looked at the documentation for JATS specifically, and did not find attributes available to the `<body>` tag that would indicate the language.  As far as I saw none of our examples uses the `@xml:lang` attribute within the body tag itself.  It is sometimes embedded in sub-elements like `<caption>`.

But I think this prompts the question of whether language as captured at the `<document>` level should be captured within `Document.json`, and I think it should, but not within body if body isn't what's declaring it.

Reply:
I'll make an issue in ingest_data_model about this.